jobstats

```diff
@@ -90,29 +91,29 @@ def job_state(USER,state,start=None,end=None):
         cmd += "-s CA"
 
 
-    print '{:>40} {}'.format("Start Date: ", startdate)
+    print('{:>40} {}'.format("Start Date: ", startdate))
     #  if end date is not Defined then only specify -S otherwise specify both -S and -E
     if end is None:
         cmd += " -S {} ".format(startdate)
     else:
         cmd += " -S {} -E {}".format(startdate,enddate)
-        print '{:>40} {}'.format("End Date: ", enddate)
+        print('{:>40} {}'.format("End Date: ", enddate))
 
-    print '{:>40} {} '.format("Job Summary by State:",state)
-    print '{:_<80}'.format('')
+    print('{:>40} {} '.format("Job Summary by State:",state))
+    print('{:_<80}'.format(''))
 
-    out = subprocess.check_output(cmd,shell="True")
+    out = subprocess.check_output(cmd, shell=True)
 
     count_cmd = cmd + " -n"
-    count_jobs = len(subprocess.check_output(count_cmd,shell="True").splitlines())
-    print out
-    print "Total Jobs: {}".format(count_jobs)
+    count_jobs = len(subprocess.check_output(count_cmd, shell=True).splitlines())
+    print(out.decode())
+    print("Total Jobs: {}".format(count_jobs))
 
 
 def main():
     """Start of main program."""
 
-    userlist = subprocess.check_output("sacctmgr list user -n --parsable2 format=user",shell="True").splitlines()
+    userlist = subprocess.check_output("sacctmgr list user -n --parsable2 format=user", shell=True).splitlines()
```

Comment:
I think there is an issue with output it returns in byte form so when i run `./jobstats -u $USER` it doesn't work i see an output list such as `b'zzhu1', b'zzq')` its a long list of user names and it doesn't match. Can you convert all user names into byte list.

Reply:
Hello @shahzebsiddiqui, thank for your vigilance :+1: , I've corrected the comportment in the following commit 
3bb63f8.

Reply:
thank you @mattdan i tested this locally and it worked out as I expected. the error is gone and it raise an error on python 2.x